pyproject.toml

```diff
@@ -42,7 +41,8 @@ pytest-mock = "*"
 mock = "*"
 mockito = "*"
 coverage = "^5.2"
-mock-firestore = "^0.7.1"
+mock-firestore = "*"
+#mock-firestore = "^0.7.1"
```

Comment:
tiny nit: commented version here, if the version is something we'd like to fallback to we can comment it next to the package? or we can remove this. 

Also, we don't have to do anything...